index.js

```diff
@@ -0,0 +1,133 @@
+const crossIcon = document.querySelector('.Cross-Icon');
+const menuItems = document.querySelector('.menu-items');
+
+function navToggle() {
+  menuItems.classList.toggle('df');
+}
+
+hamburger.addEventListener('click', navToggle);
+crossIcon.addEventListener('click', navToggle);
+
+const speakers = [
+  {
+    Image: './Images/speaker-4.png',
+    title: 'General manager',
+    name: 'Esmeralda Montana',
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit.',
+  },
+  {
+    Image: './Images/speaker-1.png',
+    title: 'Vice-Director ',
+    name: 'Becker Allison',
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit.',
+  },
+  {
+    Image: './Images/speaker-2.png',
+    title: 'General Director',
+    name: 'Frank Ghosh',
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit.',
+  },
+  {
+    Image: './Images/speaker-3.png',
+    title: 'Executive manager',
+    name: 'Claudine Besker',
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit.',
+  },
+  {
+    Image: './Images/speaker-2.png',
+    title: 'Director in Social affairs',
+    name: 'Frank Ghosh',
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit.',
+  },
+  {
+    Image: './Images/speaker-3.png',
+    title: 'Human Resources',
+    name: 'Claudine Besker',
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit.',
+  },
+  {
+    Image: './Images/speaker-3.png',
+    title: 'Human Resources',
+    name: 'Claudine Besker',
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit.',
+  },
+];
+const moreBtn = document.createElement('div');
+moreBtn.id = 'more-btn';
+const spanBtn = document.createElement('span');
+moreBtn.innerText = 'MORE';
+moreBtn.appendChild(spanBtn);
+
+const downIcon = document.createElement('img');
+downIcon.src = '/Images/down-button.png';
+moreBtn.appendChild(downIcon);
+moreBtn.style.display = 'none';
+
+const speakersContainer = document.querySelector('#speakers');
+
+for (let i = 0; i < speakers.length; i += 1) {
+  const speaker = document.createElement('div');
+  speaker.classList = 'speaker';
+  const imageDiv = document.createElement('div');
+  imageDiv.classList = 'image';
+
+  const image = document.createElement('img');
+  image.src = speakers[i].Image;
+  imageDiv.appendChild(image);
+  speaker.appendChild(imageDiv);
+
+  const details = document.createElement('div');
+  details.classList = 'Details';
+
+  const speakerName = document.createElement('h3');
+  details.appendChild(speakerName);
+
+  speakerName.innerText = speakers[i].name;
+
+  const italics = document.createElement('i');
+  italics.innerText = speakers[i].title;
+  details.appendChild(italics);
+
+  const horizontalLine = document.createElement('hr');
+  details.appendChild(horizontalLine);
+
+  const description = document.createElement('p');
+  details.appendChild(description);
+
+  description.innerText = speakers[i].description;
```

Comment:
- [ ] As requested by the previous reviewer you should make sure the speakers section matches the design on the mobile screen.

### Your Design
![image](https://user-images.githubusercontent.com/92303891/204086400-f23024cd-6d1a-4353-b87b-2238e1c9551f.png) 
### Project Design
 ![image](https://user-images.githubusercontent.com/92303891/204086410-0b972199-fc26-48ec-b067-cc5daf9ad259.png)